### ISSUE TITLE
loss function means

### ISSUE THREAD

**Issue by tangfayuan** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#155]
Hello, after the loss function is calculated, does it represent the total loss of all key points on each picture in each batch?

**Comment by Daniil-Osokin** [comment]
Yes, loss is computed for the all images in a single batch.

**Comment by zhouxin666251** [comment]
thanks a lot
